tests/integration/configs/IPFLT1B_DUCk.ini

```diff
@@ -7,6 +7,7 @@ spatial_dir = ./data/spatial
 auth_id = IPFLT1B_DUCk
 version = 1
 provider = OIB; metgenc version 1.9.0rc2
+reference_file_regex = (IPFLT1B_)(?P<granuleid>.+?(?=_)_)?(DUCk)\.txt
```

Comment:
Since each file has a different extension, all you need here is `\.txt` to differentiate it from the other data files.